android/src/main/java/org/reactnative/camera/tasks/BarCodeScannerAsyncTask.java

```diff
@@ -111,7 +176,8 @@ private BinaryBitmap generateBitmapFromImageData(byte[] imageData, int width, in
         width, // int width
```

Comment:
@bqwang91  I am working on this logic. I believe the key here is that by default the `sWidth/sHeight` should be the Android full device screen and we should expose a react native method that allows the Javascript developer the change the `sWidth` and `sHeight` value with Javascript by leveraging the react native bridge. thanks